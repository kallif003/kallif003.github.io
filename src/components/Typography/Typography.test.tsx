import * as React from "react"
import { render } from "@testing-library/react"
import {
	H1Head,
	H2Head,
	H3Head,
	H4Head,
	H5Head,
	Subtittle,
	Subtittle2,
	Body1,
	Body2,
} from "./Typography"

describe("Typography", () => {
	test("renders H1Head properly", () => {
		const { container } = render(<H1Head>Test</H1Head>)

		expect(container.firstChild).toMatchSnapshot()
	})

	it("renders H2Head properly", () => {
		const { container } = render(<H2Head>Test</H2Head>)

		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders H3Head properly", () => {
		const { container } = render(<H3Head>Test</H3Head>)

		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders H4Head properly", () => {
		const { container } = render(<H4Head>Test</H4Head>)

		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders H5Head properly", () => {
		const { container } = render(<H5Head>Test</H5Head>)

		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Subtittle properly", () => {
		const { container } = render(<Subtittle>Test</Subtittle>)

		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Subtittle 2 properly", () => {
		const { container } = render(<Subtittle2>Test</Subtittle2>)

		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Body 1 properly", () => {
		const { container } = render(<Body1>Test</Body1>)

		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Body 2 properly", () => {
		const { container } = render(<Body2>Test</Body2>)

		expect(container.firstChild).toMatchSnapshot()
	})
})
