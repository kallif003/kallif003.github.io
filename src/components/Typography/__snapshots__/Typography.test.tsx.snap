// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography renders Body 1 properly 1`] = `
<p
  class="sc-iCfMLu PkpMU"
>
  Test
</p>
`;

exports[`Typography renders Body 2 properly 1`] = `
<p
  class="sc-furwcr jWSQqj"
>
  Test
</p>
`;

exports[`Typography renders H1Head properly 1`] = `
<h1
  class="sc-bdvvtL kisRZ"
>
  Test
</h1>
`;

exports[`Typography renders H2Head properly 1`] = `
<h2
  class="sc-gsDKAQ hzsiZi"
>
  Test
</h2>
`;

exports[`Typography renders H3Head properly 1`] = `
<h3
  class="sc-dkPtRN blpyMK"
>
  Test
</h3>
`;

exports[`Typography renders H4Head properly 1`] = `
<h4
  class="sc-hKwDye fjPtml"
>
  Test
</h4>
`;

exports[`Typography renders H5Head properly 1`] = `
<h5
  class="sc-eCImPb dPnHSp"
>
  Test
</h5>
`;

exports[`Typography renders Subtittle 2 properly 1`] = `
<h2
  class="sc-gKclnd uruwL"
>
  Test
</h2>
`;

exports[`Typography renders Subtittle properly 1`] = `
<h2
  class="sc-jRQBWg ecmFyv"
>
  Test
</h2>
`;
